admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1124,6 +1134,44 @@ class AdminModuleController {
       self.updateModuleVisibility();
     });
   }
+
+  updateTotalResults() {
+    // If there are some shortlist: each shortlist count the modules on the next container.
+    const $shortLists = $('.module-short-list');
+    if ($shortLists.length > 0) {
+      $shortLists.each(function shortLists() {
+        const $this = $(this);
+        updateText(
+          $this.find('.module-search-result-wording'),
+          $this.next('.modules-list').find('.module-item').length
+        );
+      });
+
+      // If there is no shortlist: the wording directly update from the only module container.
+    } else {
+      const modulesCount = $('.modules-list').find('.module-item').length;
+      updateText($('.module-search-result-wording'), modulesCount);
+
+      if (self.currentDisplay === self.DISPLAY_LIST) {
+        $(this.addonItemListSelector).toggle(modulesCount !== (this.modulesList.length / 2));
+      } else {
+        $(this.addonItemGridSelector).toggle(modulesCount !== (this.modulesList.length / 2));
+      }
+
+      if (modulesCount === 0) {
+        $('.module-addons-search-link').attr(
+          'href',
+          `${this.bsaeAddonsUrl}search.php?search_query=${encodeURIComponent(this.currentTagsList.join(' '))}`
+        );
+      }
+    }
+
+    function updateText(element, value) {
```

Comment:
Functions must be declared as named function expressions (see [7.1](https://github.com/airbnb/javascript#functions--declarations)).

```js
const updateText = function aMoreDescriptiveAndUniqueName(element, value) { //...
```

To avoid hoisting problems, make sure to declare functions _before_ they are used (see [no-use-before-define](https://eslint.org/docs/rules/no-use-before-define)).

Reply:
Done. Thanks!